R/tm_g_association.R

```diff
@@ -305,26 +314,10 @@ srv_tm_g_association <- function(id,
       # association
       ref_class_cov <- ifelse(association, ref_class, "NULL")
 
-      teal.code::chunks_push(
-        id = "title_call",
-        expression = substitute(
-          expr = title <- new_title,
-          env = list(new_title = paste(
-            "Association",
-            ifelse(ref_class_cov == "NULL", "for", "between"),
-            paste(lapply(vars_names, function(x) {
-              if (is.numeric(ANL[[x]]) && log_transformation) {
-                varname_w_label(x, ANL, prefix = "Log of ")
-              } else {
-                varname_w_label(x, ANL)
-              }
-            }), collapse = " / "),
-            ifelse(ref_class_cov == "NULL", "", paste("and", ref_cl_lbl))
-          ))
-        )
-      )
+      print_call <- quote(print(p))
 
-      teal.code::chunks_safe_eval()
+      teal.code::eval_code(merged$anl_q_r(), substitute(expr = p <- ref_call, env = list(ref_call = ref_call)), name = "plot_call") %>%
+        teal.code::eval_code(print_call, name = "print_call")
```

Comment:
@BLAZEWIM 
When you `eval_code` you need to assign this to something to make it work like 
```r
q1 <- eval_code(<quosure>, <expression>)
```  
However, this call is something which has not been in this module before so I suggest to remove it